src/pages/UpdatesPage.tsx

```
@@ -1,77 +1,416 @@
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [modalTitle, setModalTitle] = useState('');
+  const [changedFiles, setChangedFiles] = useState<FileChange[]>([]);
+  const [isLoadingFiles, setIsLoadingFiles] = useState(false);
+
+  useEffect(() => {
+    const fetchReleases = async () => {
+      try {
+        const response = await fetch('https://api.github.com/repos/mukiwu/web-api-collection/releases');
+        const releases = await response.json();
+        
+        const formattedReleases = releases.map((release: any, index: number): Update => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)